### ISSUE TITLE
Export backup data

### ISSUE THREAD

**Issue by rdiazr** [donaldzou/WGDashboard#510]
hi all,

I have updated to newest version and i ask you, Is it posible to export backup data?

![screen](https://github.com/user-attachments/assets/ef6ba540-1adc-4905-8aec-81aa2a69ede7)

amazing job!!!!!

thanks

P.D.: i can see 2 files (wg0_20241120214039.conf  and wg0_20241120214039.sql) inside of /etc/wireguard/WGDashboard_Backup. Are these two files the backup?

**Comment by DaanSelen** [comment]
Hi there @rdiazr from what I have looked at - the backup are the files in WGDashboard_Backup in the format:
<configname>_<year><month><day><hours><minutes><seconds>.conf/.sql. in which the backup was taken.
In my case this was:

`wg0_20241120233040.conf`

**Comment by donaldzou** [comment]
> hi all,
> 
> 
> 
> I have updated to newest version and i ask you, Is it posible to export backup data?
> 
> 
> 
> ![screen](https://github.com/user-attachments/assets/ef6ba540-1adc-4905-8aec-81aa2a69ede7)
> 
> 
> 
> amazing job!!!!!
> 
> 
> 
> thanks
> 
> 
> 
> P.D.: i can see 2 files (wg0_20241120214039.conf  and wg0_20241120214039.sql) inside of /etc/wireguard/WGDashboard_Backup. Are these two files the backup?
> 
> 
> 
> 
> 
> 

Yes! Those are the backups :) and I'll implement the download through UI in the next release.

**Comment by kambiz-aries** [comment]
Hello

Thank you very much for the panel you designed and the good features you provided us, and also a very beautiful design.
The question is that after returning the backup to the new server, the backup is not taken from Schedule Jobs and it is not transferred to the new server...

How do I solve this problem?

Thank you

**Comment by DaanSelen** [comment]
> Hello
> 
> Thank you very much for the panel you designed and the good features you provided us, and also a very beautiful design. The question is that after returning the backup to the new server, the backup is not taken from Schedule Jobs and it is not transferred to the new server...
> 
> How do I solve this problem?
> 
> Thank you

Do you mean the scheduled jobs are not included?

**Comment by kambiz-aries** [comment]
Create a SQL dump of your db/wgdashboard_jobs.db file, you just need to find where it is.

Your advice was very helpful.
I copied the database file from this path (/root/WGDashboard/src/db) and restarted the panel and it was ok.
Thank you

**Comment by DaanSelen** [comment]
Did that work?

**Comment by kambiz-aries** [comment]
> Did that work?

Yes :)

**Comment by donaldzou** [comment]
This is implemented for the next release :)

**Comment by kambiz-aries** [comment]
> This is implemented for the next release :)

One of the problems of users is that they cannot see the rest of the day and their traffic consumption...
Can you consider it in the next update?
I really thank you

**Comment by DaanSelen** [comment]
You mean importing statistics as well?
